### ISSUE TITLE
[Forge 1.20.1] Adding Fluids to Coolant Config Doesn't Apply In Game

### ISSUE THREAD

**Issue by Alaric323** [Technici4n/Powah#138]
#### Version: (make sure you are on the latest version before reporting):

- Minecraft: 1.20.1
- Forge: 47.2.20
- Powah: 5.0.4, 5.0.5 doesn't seem to fix this specific issue.

#### Issue description:

Adding entries to the json5 config file under coolant_fluids does NOT add these fluids as viable options.  Water remains the only option.  Changing the sign of the entries (negative vs positive) does not help.  These entries are similar to what other modpacks I've seen use, but adapted to my current modpack with available fluids.

![image](https://github.com/Technici4n/Powah/assets/4038592/74f43221-e0d1-40de-b726-e1bb522cac5e)
![image](https://github.com/Technici4n/Powah/assets/4038592/e251251e-8acd-4986-92e1-d2c2d8892495)

#### Steps to reproduce:

1) Add entries to config
2) Config doesn't apply?

**Comment by zurgeg** [comment]
Seems to work for me

![image](https://github.com/Technici4n/Powah/assets/46549042/c76de296-285a-4797-b301-84d62c1508c3)


Config:
```json
"coolant_fluids": {
    "minecraft:water": 1,
    "estrogen:liquid_estrogen": 10
}
```

**Comment by Technici4n** [comment]
Fixed in 1.21 with the move to data maps.
